### ISSUE TITLE
False positive `eslint` errors when using this plugin with `eslint-plugin-compat`

### ISSUE THREAD

**Issue by Ilanaya** [fi3ework/vite-plugin-checker#414]
### Describe the bug

When using `vite-plugin-checker` with [eslint-plugin-compat](https://www.npmjs.com/package/eslint-plugin-compat) errors from `eslint-plugin-compat` are not recognized. Look at screenshot below.

![image](https://github.com/user-attachments/assets/57cbdbdd-fa29-4cf0-9299-e7715c58fe66)

### Reproduction

1. Install [eslint-plugin-compat](https://www.npmjs.com/package/eslint-plugin-compat)
2. Register it in eslint config, create .browserslistrc file with default configuration
3. Add some code to force an error (e.g. add `requestAnimationFrame` call which is not supported in op_mini)
4. Start the dev server with checker plugin enabled
5. Run `npm run lint` in SB reproduction

[Stack Blitz](https://stackblitz.com/edit/vite-checker-eslint-compat?file=.eslintrc.cjs,src%2FApp.tsx,.browserslistrc,package.json)

### Expected behavior

Errors in output match errors from `npm run lint` call

### System Info

```shell
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.20.3 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.2.3 - /usr/local/bin/npm
    pnpm: 8.15.6 - /usr/local/bin/pnpm
  npmPackages:
    vite-plugin-checker: latest => 0.7.0
```

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by Ilanaya** [comment]
I needed to add this to `eslint-plugin-compat` plugin configuration
```js
      browserslistOpts: {
        env: 'production',
      },
```
